include/psa/crypto_extra.h

```diff
@@ -283,9 +283,6 @@ psa_status_t psa_key_derivation(psa_key_derivation_operation_t *operation,
                                 size_t capacity);
```

Comment:
Minor: missing word in commit ac032857ed - `it could be potentially empty if MBEDTLS_MD_C` + `is defined.`